src/radio-tile/radio-tile.scss

```diff
@@ -0,0 +1,143 @@
+
+  // Rounded corners on last tile
+  &:last-of-type {
+    border-radius: 0 $iui-border-radius $iui-border-radius 0;
+  }
+
+  // All tiles shift 1px left so we don't have 2px thick dividers
+  &:not(:first-of-type) {
+    margin-left: -1px;
+  }
+
+  &-container {
+    display: inline-flex;
+    flex-wrap: wrap;
+    width: fit-content;
+    user-select: none;
+    // Create stacking context
+    position: relative;
+    z-index: 0;
+  }
+
+  &-input {
+    width: 0;
+    height: 0;
+    appearance: none;
+    opacity: 0;
+    position: absolute;
+    @include focus-visible {
+      + * {
+        outline: 2px solid var(--iui-color-foreground-primary);
+        outline-offset: -4px;
+      }
+    }
+
+    + *:hover {
+      z-index: 2;
+      border-color: rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-2));
+
+      > * {
+        fill: var(--iui-icons-color-actionable);
+      }
+    }
+    // Styles checked radio tile
+    &:checked + * {
+      padding: $iui-s - 1;
+      z-index: 3;
+      border: 2px solid var(--iui-color-foreground-primary);
+
+      > * {
+        fill: var(--iui-icons-color-primary);
+      }
+
+      &::after {
+        content: '';
+        width: $iui-icons-default;
+        height: $iui-icons-default;
+        position: absolute;
+        top: $iui-s;
+        right: $iui-s;
+        background-color: var(--iui-color-foreground-primary);
+        mask: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><path d="M6,14L0,8l2-2l4,4l8-8l2,2L6,14z" /></svg>');
+      }
+    }
+
+    // Styles disabled radio tile
+    &:disabled + * {
+      cursor: not-allowed;
+      z-index: 0;
+      border-color: var(--iui-color-background-disabled);
+      background-color: var(--iui-color-background-disabled);
+
+      &::after {
+        background-color: var(--iui-icons-color-actionable-disabled);
+      }
+
+      .iui-radio-tile-icon {
+        fill: var(--iui-icons-color-actionable-disabled);
+        filter: $iui-icons-color-multicolor-disabled;
+      }
+
+      .iui-radio-tile-label {
+        color: var(--iui-text-color-muted);
+      }
+    }
+
+    // Styles disabled checked radio tile
+    &:disabled:checked + * {
+      z-index: 3;
+      border: 2px solid rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-4));
+    }
+  }
+
+  &-icon {
+    width: $iui-icons-xl;
+    height: $iui-icons-xl;
+    display: block;
+    margin: 0 auto;
+    margin-top: round($iui-baseline * 0.5);
+    margin-bottom: $iui-baseline;
+    fill: var(--iui-icons-color);
+    @media (prefers-reduced-motion: no-preference) {
+      transition: fill $iui-speed-fast ease-out;
+    }
+  }
+
+  &-label {
+    font-size: $iui-font-size;
+    font-weight: $iui-font-weight-semibold;
+    line-height: $iui-line-height;
+    text-align: center;
+    color: var(--iui-text-color);
+  }
+
+  &-description {
+    font-size: $iui-font-size-small;
+    line-height: round($iui-baseline * 1.375);
```

Comment:
wow, `1.375` :D

Reply:
This is unchanged from the original, but yeah, i don't recall how we landed on that value.